src/utils/dictionary.ts

```diff
@@ -1,35 +1,76 @@
+import { msg } from '@lit/localize';
+
 /**
- * Mapping of PRIV Actions to corresponding descriptions.
+ * Mapping of PRIV Actions to corresponding titles.
  */
+export const ACTION_TITLES = {
+  ACCESS: () => msg('ACCESS'),
```

Comment:
Why function? `msg` works only in the `render` method?

Reply:
This is a bit complicated, but basically it is the best solution I could figure out to a limitation of lit-localize: you cannot pass a string/template variable to msg(), only a string or template literal.

What I wanted to do:

```typescript
render() {
    return html`
      <div class="prompt-heading">${this.prompt}</div>
      <div class="actions-container">
        ${this.includedActions.map(
          a =>
            html`<action-item
              action-name=${msg(ACTION_TITLES[a])}
              action-description=${msg(ACTION_DESCRIPTIONS[a])}
              ?disabled=${!enabledActions.get(a)}
            ></action-item>`
        )}
      </div>
    `;
  }
```

But you can't pass a variable to msg, so instead I was going to change ACTION_TITLES to:

```typescript
export const ACTION_TITLES = {
  ACCESS: msg('ACCESS')
  ...
}
```

But the msg function needs to execute on each render (so it can get the translated string), so this doesn't work because msg will execute only when ACTION_TITLES is first imported.

So this was the best solution I could dream up, [I posted on the lit repo](https://github.com/lit/lit/discussions/3178#discussioncomment-3233053) to see if someone had a better way but they said this was how to do it. They also explained why msg() doesn't work with a string valued variable

Reply:
do you think there's a better way?

Reply:
I think this is a reasonable solution, can't think of a better way considering the limitations.